asaps/models.py

```diff
@@ -0,0 +1,132 @@
+from asnake.client import ASnakeClient
+import importlib
+import json
+import attr
+
+
+@attr.s
+class Client:
+    authclient = attr.ib
+
+    def createclient(secfile):
```

Comment:
Let's make this the constructor method, so rename it to `def __init__(self, secfile):`